vertical-pod-autoscaler/FAQ.md

```diff
@@ -162,6 +162,14 @@ spec:
 
 and then deploy it manually if your vpa is already configured.
 
+## Can I run the VPA in an HA configuration?
+
+The VPA admission-controller can be run with multiple active Pods at any given time.
+
+Both the updater and recommender can only run a single active Pod at a time. Should you want to run a Deployment with more than one pod, it's recommended to enable lease election with the `--leader-elect=true` parameter.
```

Comment:
```suggestion
Both the updater and recommender can only run a single active Pod at a time. Should you
want to run a Deployment with more than one pod, it's recommended to enable a lease
election with the `--leader-elect=true` parameter.
```

Reply:
It looks good to me, just a small nit: Wrap the lines

Reply:
Other long lines in this file aren't wrapped though

Reply:
@adrianmoisey, I checked that 1-2 long lines are not wrapped, other than that all lines are wrapped.
wrapping the long lines will ease the PR review.

Reply:
I think these  are the lines you are talking about:

> Set the flags `--storage=prometheus` and `--prometheus-address=<your-prometheus-address>` in the deployment for the `VPA recommender`. The `args` for the container should look something like this:

> By default, the [`--min-replicas`](pkg/updater/main.go#L44) flag on the updater is set to 2. To change this, you can supply the arg in the [deploys/updater-deployment.yaml](deploy/updater-deployment.yaml) file:

It will be a great help if you can wrap these lines also.
Thanks!